Model/inverted-index.js

```diff
@@ -88,53 +102,75 @@ class Index {
     });
     this.index[fileName] = newIndex;
   }
-
+/**
+     * It returns the index of the words
+     * @method getIndex
+     * @param {Object} fileName
+     *
+     * @return {object} gets the index
+     */
   getIndex(fileName) {
-    if (Object.keys(arguments).length < 1) {
+    if (Object.keys(fileName).length < 1) {
       return this.index;
-    } else {
-      return this.index[fileName];
     }
+
+    return this.index[fileName];
   }
-  flattenSearch() {
-    for (const arg of arguments) {
-      if (arg instanceof Object && typeof arg !== 'string') {
-        for (const item in arg) {
-          if (arg.hasOwnProperty(item)) {
-            this.flattenSearch(arg[item]);
-          }
-        }
-      } else {
-        const args = arg.split(' ');
-        args.forEach((word) => {
-          this.temp_search.push(word);
-        });
-      }
-    }
+  /**
+   * @method flattenSearch
+   * @param {void} arguments of terms
+   * @return {Array} array of terms
+   */
+  flattenSearch(...args) {
```

Comment:
Instead of `args`, use descriptive variable names, based on the content of the variables